libraries/AP_Arming/AP_Arming.h

```diff
@@ -80,7 +80,8 @@ class AP_Arming {
     enum class Required {
         NO           = 0,
         YES_MIN_PWM  = 1,
-        YES_ZERO_PWM = 2
+        YES_ZERO_PWM = 2,
+        MANUAL_PASS_THROUGH = 3
```

Comment:
```suggestion
        YES_PWM_FROM_RC = 3
```

Reply:
i much prefer my name, indicating it is for MANUAL mode, and it doesn't come direct from RC anyway, it just doesn't override to zero